pkgs/by-name/fl/flarum/package.nix

```diff
@@ -14,15 +16,27 @@ php.buildComposerProject (finalAttrs: {
     hash = "sha256-kigUZpiHTM24XSz33VQYdeulG1YI5s/M02V7xue72VM=";
   };
 
+  patches = [
+    # Add useful extensions from https://github.com/FriendsOfFlarum
+    # Extensions included: fof/upload, fof/polls, fof/subscribed
+    (fetchpatch {
+      url = "https://github.com/flarum/flarum/commit/aa4c8148f5614f437c9e7be4a5364e9d3303606f.patch";
```

Comment:
Where is the PR corresponding to this commit?

Reply:
This commit comes from your fork at https://github.com/fsagbuya/flarum/commit/aa4c8148f5614f437c9e7be4a5364e9d3303606f

Therefore, I would definitely favour a commit coming straight from the official project or switch back to a custom patch.